src/map.rs

```diff
@@ -179,8 +176,11 @@ impl<K, V, S> IndexMap<K, V, S> {
     }
 
     /// Create a new map with `hash_builder`
```

Comment:
Can you add a note about the `const`ness, and that (like `new`) it does not allocate?
(ditto on the set)

Reply:
Done. I also noted `ahash` in particular because not many hash builders are const-able.